operator/pkg/cli/kafka.go

```diff
@@ -256,6 +256,15 @@ func (kc *KafkaClient) getPipelineStatus(pipelineSpec string) (*scheduler.Pipeli
 	return res, nil
 }
 
+func getPipelineNameFromHeaders(headers []kafka.Header) (string, error) {
+	for _, header := range headers {
+		if header.Key == "pipeline" {
```

Comment:
@jklaise there is a `const` section at the start of this file. Consider adding the key name `pipeline` there?

Reply:
Changed to use the existing `PipelineSpecifier` const https://github.com/SeldonIO/seldon-core/pull/4883/commits/d0e3017b80a913be2b07a7604b244788bebcd71d